features/api-layer/src/main/java/org/opennms/features/apilayer/config/SyslogMatchExtensionManager.java

```diff
@@ -75,6 +79,7 @@ protected void triggerReload() {
         LOG.debug("Syslog configuration changed. Triggering a reload.");
         try {
             syslogdConfig.reload();
+            eventConfDao.reload();
```

Comment:
I'm not sure of this one. What's the reasoning for reloading the event conf when the syslog configuration changes?

Reply:
In the issue, Alejandro reports that

```
show-event-config -u uei.opennms.org/vendor/cisco/syslog/ifDown
```

does not output the expected event configuration. The event configurations are cached in `DefaultEventConfDao`.

The `uei.opennms.org/vendor/cisco/syslog/ifDown` event is defined in `Cisco.ext.syslog.xml`, i.e. the `CiscoSyslogMatchExtension` that registers with the `SyslogMatchExtensionManager`.

Reply:
Syslog parsing rules and event definitions are distinct.

Syslog parsing rules: https://github.com/OpenNMS/alec/blob/v1.1.1/integrations/opennms/config/src/main/resources/syslog/Cisco.ext.syslog.xml

Syslog related event definitions:
https://github.com/OpenNMS/alec/blob/v1.1.1/integrations/opennms/config/src/main/resources/events/Cisco.syslog.ext.events.xml

`CiscoSyslogMatchExtension` only extends the parsing rules and has not direct impact on the event definitions.

Reply:
Ok. Will remove that line.

Reply:
Ok. Will remove that line.